scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -311,7 +311,7 @@ def build_and_test_host_config(test_root, host_config,
     # test the installation using installed cmake examples
     # TODO: enable tests for installed examples in device configurations
     # TODO: enable tests for installed makefile-based example
-    is_device_build = "cuda" in host_config
+    is_device_build = "cuda" in host_config or "rocm" in host_config
```

Comment:
This is fallout of my previous PR when I just went with rocm because that was what Spack called it. @rhornung67, do you have a preference for `hip` or `rocm`?

Reply:
well, ROCm is the software stack/compiler, HIP is the programming model. Since we use CUDA for the nvcc compiler, let's use hip here for consistency.